clojure.behaviors

```diff
@@ -97,4 +97,18 @@
 
  [:sidebar.docs.search :lt.plugins.clojure/clj-doc-search]
  [:sidebar.docs.search :lt.plugins.clojure/cljs-doc-search]
+
+ ; collapsible stacktrace
+ [:editor.clj :lt.plugins.error-proposal/expandable-exceptions]
```

Comment:
change this to use tag :editor.clj.common so that the behavior becomes available to cljs too.